### PR TITLE
added myself as participant

### PR DESCRIPTION
**Pull request by WolverienG6**

### CODE REVIEW

**Comment by dojutsu-user** [comment]
Please update/sync this branch with `master` branch.

File: Data/participants.json [hunk 1]
```diff
@@ -9,3 +9,14 @@
     }
```
Comment (rishabhshuklax) [inline Data/participants.json:9]:
```suggestion
},
{
"name":......
}
```

**Review by rishabhshuklax — CHANGES_REQUESTED** [review]
don't add new "participant" key do like this way

**Comment by rishabhshuklax** [comment]
Due to your inactivity, I am closing this PR.

**Pull request closed without merging**